build-tools/create-packs/linux-README.md

```diff
@@ -0,0 +1 @@
+Please note that this package is not officially supported, and has not received extensive testing.
```

Comment:
Should we put this on the package description, too?
https://github.com/xamarin/xamarin-android/blob/df667b079fdccb9a263d5c12467eef7f2fa0ff55/build-tools/create-packs/Microsoft.Android.Sdk.proj#L15

Reply:
I added this to the description as well.